### PR TITLE
Sums

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p className="paragraph">
+    {'Sum of '}
+    {a}
+    {' and '}
+    {b}
+    {' is '}
+    {a + b}
+  </p>
```
Comment (vlad-tuzenko) [inline src/components/Sum.jsx:12]:
```
<p>
{`Sum of ${a} and ${b} is ${sum}`}
</p>
```
I find this code a little more readable.

**Review by vlad-tuzenko — APPROVED** [review]
Sweet design 👍

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Comment (aionova) [inline src/components/Sum.jsx:1]: we usually create a folder for each component and place its jsx and scss files these. E.g. components/Sum/Sum.jsx

**Review by aionova — APPROVED** [review]
Nice!

**Pull request closed without merging**